### PR TITLE
Fixed basic formatting using yq

### PR DESCRIPTION
**Pull request by vigneshk-tw**
## Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by sangeet-joy-tw — APPROVED** [review]

File: xero-finance.yaml [hunk 1]
```diff
@@ -2458,13 +2364,12 @@ components:
           x-is-money: true
         indicativeEndBalance:
           type: number
-          description: |
```
Comment (manishT72) [inline xero-finance.yaml:2461]: what is this | char? Did it had any significance

**Review by manishT72 — CHANGES_REQUESTED** [review]

File: xero-app-store.yaml [hunk 2]
```diff
@@ -30,57 +30,7 @@ paths:
...
-                          "startDate": "2021-07-20T03:13:48",
-                          "status": "ACTIVE"
-                        }
+              example: {"currentPeriodEnd": "2021-07-20T03:13:48", "endDate": "2021-07-20T03:13:48", "id": "01b5a6f4-8936-4bfa-b703-830702312b87", "testMode": true, "organisationId": "fdc5be44-9b3e-4ebb-a0e9-11b9737f9a28", "plans": [{"id": "f617dd59-462f-46a1-9519-1765fd38b160", "name": "Small", "status": "ACTIVE", "subscriptionItems": [{"endDate": "2021-07-20T03:13:48", "id": "c7336bf6-8a47-4f13-9fc0-82420e6922c8", "testMode": true, "price": {"amount": 50, "currency": "AUD", "id": "31acefbe-bdb7-4329-84d6-51e9afd95327"}, "product": {"id": "56d66073-ff78-497b-a726-ca9d56fdafa3", "name": "Small", "type": "FIXED"}, "quantity": 1, "startDate": "2021-07-20T03:13:48"}, {"endDate": "2021-07-20T03:13:48", "id": "1a25bc96-141a-4e42-a0fa-55ed51844da4", "testMode": true, "price": {"amount": 0.20, "currency": "AUD", "id": "15c65048-1b48-4d9f-81c0-4e1cd3a4f49c"}, "product": {"id": "e8d8ac02-01b3-47f7-8a22-99cfe9035cf7", "name": "Texts", "type": "METERED", "usageUnit": "txt"}}]}], "startDate": "2021-07-20T03:13:48", "status": "ACTIVE"}
```
[the remark this answers is not in the document]
Comment (vigneshk-tw) [reply xero-app-store.yaml:33]: if we pretty print it is removing all the curly braces and quotes. i'll check if the SDK generation and PRISM works fine and confirm

File: xero-finance.yaml [hunk 3]
```diff
@@ -1040,13 +1000,7 @@ paths:
         - Finance
       operationId: getBankStatementAccounting
       summary: Get Bank Statement Accounting
-      description: >-
```
[the remark this answers is not in the document]
Comment (vigneshk-tw) [reply xero-finance.yaml:1043]: This syntax is used for multi line handling in open API. both this results in same output when generating the code

File: xero-app-store.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vigneshk-tw) [reply xero-app-store.yaml]: verified the SDK generation and no impact in code because of this change

File: xero-finance.yaml (hunk 1 above)
Comment (vigneshk-tw) [reply xero-finance.yaml:2461]: yq automatically formats by adding \n at teh end replacing |

**Review by manishT72 — APPROVED** [review]

**Pull request merged**